app/assets/javascripts/profile/profile.coffee

```diff
@@ -0,0 +1,70 @@
+profile = angular.module("Profile",["SharedServices", "ProfilePageServices"])
+
+profile.config(["$routeProvider", ($routeProvider) ->
+    $routeProvider.when("/profile/:id", {
+        templateUrl: "/assets/profile.html",
+        controller: "ProfileController"
+    })
+]).controller("ProfileController", ["$http", "$location", "$scope", "$routeParams", "SharedRequests", "ProfilePageData", ($http, $location, $scope, $routeParams, SharedRequests, ProfilePageData) ->
+    # moduleState can be either "view" or "edit"
+    $scope.moduleState = "view"
+
+    $scope.loggedInUID = 1 # FIXME Hardcode UID as 1 for now
+    $scope.profileUID = $routeParams.id
+
+    $scope.profile = ProfilePageData
+
+    # For radio buttons
+    $scope.spectrumOptions = {
+        conservative: {
+            "id": 1
+            "value": "Conservative"
+        },
+        moderately_conservative: {
+            "id": 2
+            "value": "Moderately Conservative"
+        },
+        moderate: {
+            "id": 3
+            "value": "Moderate"
+        },
+        moderately_liberal: {
+            "id": 4
+            "value": "Moderately Liberal"
+        },
+        liberal: {
+            "id": 5
+            "value": "Liberal"
+        }
+    }
+
+    $scope.can_edit = () ->
+        return 1==1
```

Comment:
Assuming you're referring to the `1==1` that's there because we don't have a way of knowing who's logged in until that's implemented, so that's something that's still "todo" (as listed @ the top of the PR).  So right now A can edit B's profile, but that won't be the case once knowledge of who's logged in is implemented and Angular can tell that A is logged in and is viewing B's profile.